src/dmdevfs.c

```diff
@@ -404,8 +469,8 @@ dmod_dmfsi_dif_api_declaration( 1.0, dmdevfs, int, _direxists, (dmfsi_context_t
         return 0;
     }
     
-    // TODO: Implement directory existence check
-    return 0;
+    // Only root directory exists
+    return is_root_path(path) ? 1 : 0;
```

Comment:
to nie jest prawda, bo jeśli dla jakiegoś drivera został stworzony katalog np `/dev/uart0/` to też musimy go wspierać